wagtail_wordpress_import/importers/import_hooks.py

```diff
@@ -0,0 +1,45 @@
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+
+class ItemsCache:
+    """Store the WordPress XML item tags.
+    These are items tags that don't represent a page in the XML file.
+    They are temporarily stored and used at the end of the import process.
+    """
+
+    def __init__(self):
+        """For each value in settings.WORDPRESS_IMPORT_HOOKS_ITEMS_TO_CACHE
+        create the class attribute.
+        """
+
+        for hook in getattr(
+            settings, "WORDPRESS_IMPORT_HOOKS_ITEMS_TO_CACHE", {}
+        ).keys():
+            setattr(self, hook, [])
+
+    def process(self, imported_pages):
+        """Run all hooks in the config for each page.
+
+        Each function defined in the config key "FUNCTION" will be run on each page.
+        """
+        registry = self._get_hook_handler_data()
+        for page in imported_pages:
+            for hook, (func, data_tag) in registry.items():
+                func(page, data_tag, self.__dict__)
```

Comment:
**Suggestion**: Now we've merged the multiple-handlers changes, I would suggest that the data to be passed to the registered function should only take the data that it asks for. This would mean changing the example functions, too.

```suggestion
                func(page, data_tag, getattr(self, hook))
```

Reply:
The example functions in the docs are working locally for me with no changes.